pkg/cloudprovider/cloudprovider.go

```diff
@@ -10,8 +10,15 @@ import (
 const (
 	failureDomainZoneKey   = v1.LabelTopologyZone
 	failureDomainRegionKey = v1.LabelTopologyRegion
```

Comment:
Can we remove these constants now and just use `v1.LabelTopologyZone` directly in the tests?